### PR TITLE
Import mentors

### PR DESCRIPTION
**Pull request by elft3r**
This PR should allow to import  the `mentors.json` file as specified in #26.

However, during the cleanup of the script I broke something and now the users are no longer written to mongo. As I'm still quite new to javascript, I could not yet figure out, why the `user.save()` function is no longer working properly.

I hope someone can point me to the problem I'm facing.

Thanks

### CODE REVIEW

File: scripts/import-mentors.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      title: mentor.title,
+      description: mentor.description,
+      country: mentor.country,
+      spokenLanguages: mentor.spokenLanguages,
+      tags: mentor.tags,
+      roles: [Role.MENTOR],
+      channels: mentor.channels,
+    })
+
+    user.save(function (err, results) {
```
Comment (brentmclark) [inline scripts/import-mentors.ts]: This is an asynchronous command; you probably need to `await` here.

File: scripts/import-mentors.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  console.log("Connect to database")
+  mongoose.connect(process.env.MONGO_DATABASE_URL, { useNewUrlParser: true });
+
+  var User = mongoose.model('User', UserSchema);
+
+  console.log("Store mentors to database")
+  for (const mentor of mentors) {
+    console.log(`Process mentor '${mentor.id}'`)
+
+    var user = new User({
```
Comment (crysfel) [inline scripts/import-mentors.ts]: Before creating a new user make sure the user doesn't exist in the database, maybe the current mentor is already there (while I was testing I got my username twice in the database)

File: scripts/import-mentors.ts (hunk 1 above)
Comment (crysfel) [reply scripts/import-mentors.ts]: After adding the `await` and removing this callback I started seeing the users added to the database.

Comment (elft3r) [reply scripts/import-mentors.ts]: Thanks for your help, with these changes, I was able to make it work :-)

File: scripts/import-mentors.ts (hunk 2 above)
Comment (elft3r) [reply scripts/import-mentors.ts]: Very good point. The new code first checks whether the mentor is already in the database.

File: scripts/import-mentors.ts [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+      var newUser = new User({
+        name: mentor.name,
+        email: mentor.id,
+        avatar: mentor.avatar,
+        title: mentor.title,
+        description: mentor.description,
+        country: mentor.country,
+        spokenLanguages: mentor.spokenLanguages,
+        tags: mentor.tags,
+        roles: [Role.MENTOR],
```
Comment (crysfel) [inline scripts/import-mentors.ts]: We also need to include the `Role.MEMBER` in this array.

**Pull request merged**